erase-install.sh

```diff
@@ -180,10 +180,15 @@ dialog_power_title_de="Warten auf AC-Netzteil"
 dialog_power_title_nl="Wachten op Stroomadapter"
 dialog_power_title_fr="En attente de l'alimentation secteur"
 
-dialog_power_desc_en="Please connect your computer to power using an AC power adapter. This process will continue once AC power is detected."
-dialog_power_desc_de="Bitte schließen Sie Ihren Computer mit einem AC-Netzteil an das Stromnetz an. Dieser Prozess wird fortgesetzt, sobald die AC-Stromversorgung erkannt wird."
-dialog_power_desc_nl="Sluit uw computer aan met de stroomadapter. Zodra deze is gedetecteerd gaat het proces verder"
-dialog_power_desc_fr="Veuillez connecter votre ordinateur à un adaptateur secteur. Ce processus se poursuivra une fois que l'alimentation secteur sera détectée."
+dialog_power_desc_en="Please connect your computer to power using an AC power adapter. This process will continue if AC power is detected within the next:"
+dialog_power_desc_de="Bitte schließen Sie Ihren Computer mit einem AC-Netzteil an das Stromnetz an. Dieser Prozess wird fortgesetzt, sobald die AC-Stromversorgung erkannt wird innerhalb der nächsten:"
```

Comment:
The sentence is a bit tricky to do grammatically in German as the second verb should be at the end (after the time). Let's try:

```
Bitte schließen Sie Ihren Computer mit einem AC-Netzteil an das Stromnetz an. Dieser Prozess wird fortgesetzt, sobald die AC-Stromversorgung innerhalb der folgende Zeitdauer erkannt wird:
```

Reply:
(I don't mind making these changes after the merge)

Reply:
Thanks, Obi-@grahampugh!